ansible/roles/haproxy/defaults/main.yml

```diff
@@ -0,0 +1,205 @@
+---
+# .. vim: foldmarker=[[[,]]]:foldmethod=marker
+
+# Copyright (C) 2020 Innobyte Alin Alexandru <https://www.innobyte.com/>
+# Copyright (C) 2020 Innobyte Bechea Leonardo <https://www.innobyte.com/>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# debops.haproxy default variables
+# ================================
+
+# .. contents:: Sections
+#    :local:
+#
+# .. include:: ../../../../includes/global.rst
```

Comment:
This `global.rst` is something we're trying to move away from.

If you're not using any of the refs contain in there you can just remove it. Otherwise, just inline the refs you need.